ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/ipv6_static_routes.schema.yml

```diff
@@ -0,0 +1,43 @@
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  ipv6_static_routes:
+    type: list
+    display_name: IPv6 Static Routes
+    items:
+      type: dict
+      keys:
+        vrf:
+          type: str
+          display_name: VRF Name
```

Comment:
We should add a description to capture the behavior of "default" VRF: `if vrf_name = default the route will be placed in the GRT `

Reply:
IMO This description in not very clear, and to me GRT is not a common term.
Also I think it is a bit superfluous to describe that when you set 'default' VRF the route ends up in 'default' VRF.